Gems/GradientSignal/Code/Source/Components/ImageGradientComponent.cpp

```diff
@@ -288,13 +479,65 @@ namespace GradientSignal
                 // Flip the y because images are stored in reverse of our world axes
                 y = (height - 1) - y;
 
-                return AZ::RPI::GetImageDataPixelValue<float>(m_imageData, imageDescriptor, x, y);
+                // By default, we will query from the R channel of the image
+                auto channel = ChannelToUse::R;
+
+                // Advanced handling of channel and scale
+                if (m_configuration.m_advancedMode)
+                {
+                    channel = m_configuration.m_channelToUse;
```

Comment:
If advanced mode is false, it should just use `ChannelToUse::Red`, since the channel drop down is in the advanced section. But I can make it so that the channel along with the multiplier/offset (mentioned below) are calculated when once when the asset is ready.